rdrf/rdrf/settings.py

```diff
@@ -337,6 +337,22 @@
 # #
 LOG_DIRECTORY = env.get('log_directory', os.path.join(WEBAPP_ROOT, "log"))
 
+# Log handlers
+DEFAULT_LOG_HANDLER = ['console', 'file']
+MAILADMIN_LOG_HANDLER = ['mail_admins']
+COMMAND_LOG_HANDLER = ['shell', 'admin_command_file']
+IMPORT_LOG_HANDLER = ['console_simple']
+
+# Add the syslog handler when syslog is enabled.
+SYSLOG_ENABLED = env.get("syslog_enabled", False)
+SYSLOG_ADDRESS = env.get("syslog_address", "172.21.0.1")
```

Comment:
Don't think we should have a default ip here. 

>

Reply:
this is the default docker host ip, I would argue that it is actually a good idea to leave it here for DX as most dev won't have the pain to find out for themself when they install rsyslog in their host.